diff --git a/node_modules/framer-motion/dist/framer-motion.d.ts b/node_modules/framer-motion/dist/framer-motion.d.ts
index f98d835..58d02cc 100644
--- a/node_modules/framer-motion/dist/framer-motion.d.ts
+++ b/node_modules/framer-motion/dist/framer-motion.d.ts
@@ -1581,7 +1581,7 @@ export declare function isValidMotionProp(key: string): boolean;
 /**
  * @internal
  */
-declare interface Just {
+export declare interface Just {
     type: "just";
     to?: number | string | ValueTarget;
     from?: number | string;
diff --git a/node_modules/framer-motion/dist/framer-motion.dev.js b/node_modules/framer-motion/dist/framer-motion.dev.js
index 263c5a8..99a9e8f 100644
--- a/node_modules/framer-motion/dist/framer-motion.dev.js
+++ b/node_modules/framer-motion/dist/framer-motion.dev.js
@@ -6721,6 +6721,7 @@
     exports.useTapGesture = useTapGesture;
     exports.useTransform = useTransform;
     exports.useViewportScroll = useViewportScroll;
+    exports.useDrag = useDrag;
 
     Object.defineProperty(exports, '__esModule', { value: true });
 
diff --git a/node_modules/framer-motion/dist/framer-motion.es.js b/node_modules/framer-motion/dist/framer-motion.es.js
index f3a864c..ab70651 100644
--- a/node_modules/framer-motion/dist/framer-motion.es.js
+++ b/node_modules/framer-motion/dist/framer-motion.es.js
@@ -4587,4 +4587,4 @@ var AnimatePresence = function (_a) {
         : childrenToRender.map(function (child) { return cloneElement(child); })));
 };
 
-export { AnimatePresence, AnimationControls, MotionContext, MotionPluginContext, MotionPlugins, MotionValue, Point, UnstableSyncLayout, animationControls, createMotionComponent, isValidMotionProp, motion, motionValue, transform, unwrapMotionValue, useAnimatedState, useAnimation, useCycle, useDomEvent, useExternalRef, useGestures, useInvertedScale, useMotionValue, usePanGesture, useSpring, useTapGesture, useTransform, useViewportScroll };
+export { AnimatePresence, AnimationControls, MotionContext, MotionPluginContext, MotionPlugins, MotionValue, Point, UnstableSyncLayout, animationControls, createMotionComponent, isValidMotionProp, motion, motionValue, transform, unwrapMotionValue, useAnimatedState, useAnimation, useCycle, useDomEvent, useExternalRef, useGestures, useInvertedScale, useMotionValue, usePanGesture, useSpring, useTapGesture, useTransform, useViewportScroll, useDrag };
