diff --git a/node_modules/react-refresh/cjs/react-refresh-runtime.development.js b/node_modules/react-refresh/cjs/react-refresh-runtime.development.js
index b0ad68b..4cb15e8 100644
--- a/node_modules/react-refresh/cjs/react-refresh-runtime.development.js
+++ b/node_modules/react-refresh/cjs/react-refresh-runtime.development.js
@@ -521,6 +521,10 @@ function isLikelyComponentType(type) {
       case 'object':
         {
           if (type != null) {
+            if (type['__react_refresh_keep']) {
+              return true
+            }
+
             switch (type.$$typeof) {
               case REACT_FORWARD_REF_TYPE:
               case REACT_MEMO_TYPE:
diff --git a/node_modules/react-refresh/runtime.js b/node_modules/react-refresh/runtime.js
index 28e3b0c..5bbd389 100644
--- a/node_modules/react-refresh/runtime.js
+++ b/node_modules/react-refresh/runtime.js
@@ -1,7 +1,3 @@
 'use strict';
 
-if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./cjs/react-refresh-runtime.production.min.js');
-} else {
-  module.exports = require('./cjs/react-refresh-runtime.development.js');
-}
+module.exports = require('./cjs/react-refresh-runtime.development.js');
\ No newline at end of file
