diff --git a/node_modules/webpack-hot-middleware/client.js b/node_modules/webpack-hot-middleware/client.js
index abc95ed..6d72ef5 100644
--- a/node_modules/webpack-hot-middleware/client.js
+++ b/node_modules/webpack-hot-middleware/client.js
@@ -89,6 +89,11 @@ function EventSourceWrapper() {
     source.onopen = handleOnline;
     source.onerror = handleDisconnect;
     source.onmessage = handleMessage;
+
+    window.addEventListener('beforeunload', () => {
+      console.log('unloading webpack-hot-middleware eventsource');
+      source.close();
+    });
   }
 
   function handleOnline() {
