diff --git a/node_modules/webpack-hot-middleware/client.js b/node_modules/webpack-hot-middleware/client.js
index abc95ed..2e1fea8 100644
--- a/node_modules/webpack-hot-middleware/client.js
+++ b/node_modules/webpack-hot-middleware/client.js
@@ -89,6 +89,10 @@ function EventSourceWrapper() {
     source.onopen = handleOnline;
     source.onerror = handleDisconnect;
     source.onmessage = handleMessage;
+
+    window.addEventListener('beforeunload', () => {
+      source.close()
+    })
   }
 
   function handleOnline() {
